example/lib/main.dart

```diff
@@ -78,49 +88,87 @@ class _MyHomePageState extends State<MyHomePage> {
         title: Text(widget.title),
       ),
       body: LazyLoadScrollView(
-        isLoading: isLoadingVertical,
-        onEndOfPage: () => _loadMoreVertical(),
-        child: Scrollbar(
-          child: ListView(
-            children: [
-              Padding(
-                padding: const EdgeInsets.all(8.0),
-                child: Text(
-                  'Nested horizontal ListView',
-                  textAlign: TextAlign.center,
+        onLoadAfter: _loadMoreVertical,
+        controller: scrollController,
+        builder: (context, isLoadingAfter, _) {
+          // FIXME: I don't think this is true
+          // Scrollbar eats notifications ?? so we'll just use a scroll controller
```

Comment:
Could you elaborate on the concern here?

Reply:
Ah yeah, this was weird. So I think even if I set the predicate to the right layer where the descendant scrollable was, I couldn't get the thing to work (without using a scrollcontroller). I actually think the `Scrollbar` is handling the notifications and not allowing them to bubble (by returning false in their `NotificationListener` onNotification). Can you confirm that it worked previously with your implementation (wrt Scrollbar specifically)?